spec/requests/api_stories/move_workflows/1_police_to_police/1_1_person_with_police_national_computer_spec.rb

```diff
@@ -150,6 +150,9 @@
     police1
     police2
 
+    # Assign police1 location to Serco supplier
+    create :supplier_location, location: police1, supplier: serco_supplier, effective_from: Date.new(1900, 1, 1)
```

Comment:
It's cleaner to just leave `effective_from` as `nil`. The behaviour of all the supplier chooser and supplier location models is well covered elsewhere and it's perfectly fine to create a supplier location with nil effective dates - makes it much easier to test.

Reply:
👍